### PR TITLE
ASG nodes initialisation

### PR DESCRIPTION
**Pull request by Shokubeni**

### CODE REVIEW

File: component/kubernetes/module/common/outputs.tf [hunk 1]
```diff
@@ -0,0 +1,7 @@
+output "system_commands" {
+  value = "${module.ssm_command.sustem_commands}"
```
Comment (NikitaZhevnitskiy) [inline component/kubernetes/module/common/outputs.tf]: typo

File: component/kubernetes/module/common/module/ssm_command/outputs.tf [hunk 2]
```diff
@@ -0,0 +1,5 @@
+output "sustem_commands" {
```
Comment (NikitaZhevnitskiy) [inline component/kubernetes/module/common/module/ssm_command/outputs.tf]: typo

File: component/kubernetes/module/common/outputs.tf (hunk 1 above)
Comment (Shokubeni) [reply component/kubernetes/module/common/outputs.tf]: done

File: component/kubernetes/module/common/module/ssm_command/outputs.tf (hunk 2 above)
Comment (Shokubeni) [reply component/kubernetes/module/common/module/ssm_command/outputs.tf]: done

**Pull request merged**